src/main/java/com/ibm/crypto/plus/provider/ECUtils.java

```diff
@@ -254,4 +257,36 @@ static String bytesToHex(byte[] bytes) {
         return new String(hexChars);
     }
 
+    /**
+     * Check an ECPrivateKey to make sure the scalar value is within the
+     * range of the order [1, n-1].
+     * @param prv the private key to be checked.
+     * @return the private key that was evaluated.
+     * @throws InvalidKeyException if the key's scalar value is not within
+     *      the range 1 <= x < n where n is the order of the generator.
+     */
+    public static java.security.interfaces.ECPrivateKey checkPrivateKey(java.security.interfaces.ECPrivateKey prv)
```

Comment:
Can we make use of this same function in the sun.security.util package in the class ECUtil.java? Then in this case we dont have to maintain a copy of this function in OpenJCEPlus.

Reply:
Replaced with sun.security.util package